packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -207,7 +203,7 @@ export class SchemaConverter implements SimpleDbSchemaConverter {
     return queuesStore.loadAll().next(queues => {
       return PersistencePromise.forEach(queues, (queue: DbMutationQueue) => {
         const range = IDBKeyRange.bound(
-          [queue.userId, BATCHID_UNKNOWN],
+          [queue.userId, Number.MIN_SAFE_INTEGER],
```

Comment:
Maybe put BATCHID_UNKNOWN in core/types so that this can remain a reference to that constant?

Reply:
Done